src/DataLayer/dataSources/DataSource.ts

```diff
@@ -1,25 +1,95 @@
 // (C) 2007-2018 GoodData Corporation
-import { AFM } from '@gooddata/typings';
+import { AFM, Execution } from '@gooddata/typings';
 import * as stringify from 'json-stable-stringify';
 import { IDataSource } from '../interfaces/DataSource';
 
 export {
     IDataSource
 };
 
+export const SERVER_SIDE_COLUMN_LIMIT = 1000;
```

Comment:
Are you sure its a 1000 and not 500 ?

Reply:
https://github.com/gooddata/gooddata-js/blob/master/src/execution/execute-afm.ts#L9 see there is a different limit